lib/exercism/five_a_day_count_exercises.rb

```diff
@@ -0,0 +1,33 @@
+class Five
```

Comment:
This file was deleted 2 days ago, it snuck in the Incident (merge/squash mess).
And this class was renamed 3 days ago. 
How should we proceed?